[BUG] Running quarkus:dev fails with date format error

The latest commit on `master` gives me this error when running `mvn clean compile quarkus:dev`:

```
$ mvn clean compile quarkus:dev
[INFO] Scanning for projects...
[INFO]
[INFO] ----------< org.quarkus.samples.petclinic:quarkus-petclinic >-----------
[INFO] Building quarkus-petclinic 1.0.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]

[...]

2023-01-24 13:28:48,494 INFO  [io.qua.dat.dep.dev.DevServicesDatasourceProcessor] (build-40) Dev Services for the default datasource (postgresql) started - container ID is 98678c40453e
2023-01-24 13:28:52,317 INFO  [io.qua.kub.cli.dep.DevServicesKubernetesProcessor] (build-52) Dev Services for Kubernetes started. Other Quarkus applications in dev mode will find the cluster automatically.
2023-01-24 13:28:52,319 INFO  [io.qua.dep.dev.IsolatedDevModeMain] (main) Attempting to start live reload endpoint to recover from previous Quarkus startup failure
2023-01-24 13:28:52,497 ERROR [io.qua.dep.dev.IsolatedDevModeMain] (main) Failed to start quarkus: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.qute.deployment.QuteProcessor#processTemplateErrors threw an exception: io.quarkus.qute.TemplateException: Found incorrect expressions (4):
	[1] ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[2] ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[3] createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[4] createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method

	at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:833)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more

	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:335)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:252)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.createInitialRuntimeApplication(AugmentActionImpl.java:60)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.firstStart(IsolatedDevModeMain.java:86)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:447)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:59)
	at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:149)
	at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:104)
	at io.quarkus.deployment.dev.DevModeMain.start(DevModeMain.java:131)
	at io.quarkus.deployment.dev.DevModeMain.main(DevModeMain.java:62)
Caused by: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
	[error]: Build step io.quarkus.qute.deployment.QuteProcessor#processTemplateErrors threw an exception: io.quarkus.qute.TemplateException: Found incorrect expressions (4):
	[1] ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[2] ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[3] createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[4] createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method

	at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:833)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-ddoperty/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more

	at io.quarkus.builder.Execution.run(Execution.java:123)
	at io.quarkus.builder.BuildExecutionBuilder.execute(BuildExecutionBuilder.java:79)
	at io.quarkus.deployment.QuarkusAugmentor.run(QuarkusAugmentor.java:160)
	at io.quarkus.runner.bootstrap.AugmentActionImpl.runAugment(AugmentActionImpl.java:331)
	... 9 more
Caused by: io.quarkus.qute.TemplateException: Found incorrect expressions (4):
	[1] ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[2] ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[3] createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
	[4] createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method

	at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:238)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at java.base/java.lang.Thread.run(Thread.java:833)
	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:45:17 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: ownerDetails.html:65:19 - {visit.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:20:13 - {pet.birthDate.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more
	Suppressed: io.quarkus.qute.TemplateException: createOrUpdateVisitForm.html:77:13 - {v.date.format('yyyy-MM-dd')}: Property/method [format('yyyy-MM-dd')] not found on class [java.time.LocalDate] nor handled by an extension method
		at io.quarkus.qute.TemplateException$Builder.build(TemplateException.java:169)
		at io.quarkus.qute.deployment.QuteProcessor.processTemplateErrors(QuteProcessor.java:215)
		... 11 more



--
Tests paused
Press [r] to resume testing, [o] Toggle test output, [:] for the terminal, [h] for more options>
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  14.640 s
[INFO] Finished at: 2023-01-24T13:28:56Z
[INFO] ------------------------------------------------------------------------
%
```

--- a/src/main/java/org/quarkus/samples/petclinic/Temporals.java
+++ b/src/main/java/org/quarkus/samples/petclinic/Temporals.java
@@ -1,0 +1,22 @@
+package org.quarkus.samples.petclinic;
+
+import io.quarkus.qute.TemplateExtension;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This is a temporary hack around
+ * <a href="https://github.com/redhat-developer-demos/quarkus-petclinic/issues/7">this</a> issue.
+ * <p>
+ * The problem appears to be M1 specific and is being
+ * <a href="https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/petclinic">looked</a> into by the Quarkus devs.
+ */
+@TemplateExtension
+public class Temporals {
+
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+
+    public static String format(LocalDate localDate) {
+        return localDate.format(FORMATTER);
+    }
+}

--- a/src/main/resources/templates/createOrUpdateVisitForm.html
+++ b/src/main/resources/templates/createOrUpdateVisitForm.html
@@ -17,7 +17,7 @@
     </thead>
     <tr>
         <td>{pet.name}</td>
-        <td>{pet.birthDate.format('yyyy-MM-dd')}</td>
+        <td>{pet.birthDate.format()}</td>
         <td>{pet.type.name}</td>
         <td>{pet.owner.firstName} {pet.owner.lastName}</td>
     </tr>
@@ -74,7 +74,7 @@
     <tr>
         {#if visit}
         {#for v in pet.visits}
-        <td>{v.date.format('yyyy-MM-dd')}</td>
+        <td>{v.date.format()}</td>
         <td>{v.description}</td>
         {/for}
         {/if}

--- a/src/main/resources/templates/ownerDetails.html
+++ b/src/main/resources/templates/ownerDetails.html
@@ -42,7 +42,7 @@
             <dt>{msg:name}</dt>
             <dd>{pet.name}</dd>
             <dt>{msg:birthdate}</dt>
-            <dd>{pet.birthDate.format('yyyy-MM-dd')}</dd>
+            <dd>{pet.birthDate.format()}</dd>
             <dt>{msg:type}</dt>
             <dd>{pet.type.stringify}</dd>
           </dl>
@@ -62,7 +62,7 @@
             </thead>
             {#for visit in pet.visits}
             <tr>
-              <td>{visit.date.format('yyyy-MM-dd')}</td>
+              <td>{visit.date.format()}</td>
               <td>{visit.description ?: ''}</td>
             </tr>
             {/for}

